package.json

```diff
@@ -13,13 +17,6 @@
     "test:ui": "cypress run",
     "test:ui:open": "cypress open"
   },
-  "workspaces": {
-    "packages": [
-      "apps/*",
-      "libs/*",
-      "test"
-    ]
-  },
```

Comment:
This project uses yarn workspaces since it is a monorepo of 4 projects, 2 public (heimdall server and heimdall lite) and 2 private (Server E2E tests and Heimdall Interfaces). Removing the workspaces will cause issues with the build.

Reply:
this was removed to avoid a new bug we encountered. For any node application using the vscode workspace feature the cloud foundry buildpack will fail to parse the package.json file and will throw `**ERROR** Failed parsing package.json: json: cannot unmarshal object into Go struct field .workspaces of type []string`

Will revert its removal as a new commit

Reply:
This shouldn't be a vscode workspace, this is utilizing yarn workspaces https://yarnpkg.com/features/workspaces. Any vscode workspace related information should be stored in the `heimdall2.code-workspace` file.